usaspending_api/api_docs/markdown/using_the_api.md

```diff
@@ -17,19 +17,19 @@
 
 # Using the USAspending Application Program Interface (API) <a name="introduction"></a>
 
-The USAspending API allows the public to access data published via the DATA Act Data Broker or via USAspending.
+The USAspending API allows the public to access data published via the DATA Act Data Broker and USAspending.
 
 This guide is intended for users who are already familiar with APIs. If you're not sure what _endpoint_ means, and what `GET` and `POST` requests are, you'll probably find the [introductory tutorial](/docs/intro-tutorial) more useful.
 
-The USASpending API is in V2. V1 endpoints are currently deprecated.
+The USAspending API is in V2. V1 endpoints are currently deprecated.
```

Comment:
Was there specific direction to write they "are currently deprecated"? There are still many working and in-use v1 endpoints. Could change to "will be deprecated in the near future"

Reply:
Ignore. This can remain as-is